### ISSUE TITLE
Issue with upgrading an item

### ISSUE THREAD

**Issue by Spydoc** [CookieMonsterDev/titans-league#34]
**Description**:

I'm not sure if there are level penalties, which cause your stats from upgrading to get reduced when you're leveling up your character, but if that's the case, then this report is invalid.
An item isn't getting the stats from upgrading with a level 1 stat. Upon leveling the stat to level 2, it shows that it gives +1 of the desired stat but when it's on level 1, it's getting 0.

**Current behaviour**:

Upon upgrading my chest piece with +1 agility, it got the stat but at a certain point it lost it.

**Expected behaviour**:

The item should keep the stats from upgrading.

**Steps to reproduce the problem**:

Upgrade an item once and magnetize it. In the beginning my item did have +2 agility from the upgrade and the magnetized robe but at some point it lost the stat from upgrading. As you can see in the screenshot below, it shows that it has level 1 agility but it's actually giving 0 agility.

**Any other useful links here**:

https://imgur.com/a/ka1DlAo

**Comment by Spydoc** [comment]
Update to this report:

I have an issue with two items currently.
*The Titan Staff was upgraded with level 1 stamina and I magnetized a two-handed mace (Stone Hammer) on to it. The mace has 16 stamina, so the staff should have 17 stamina but instead it has only 1.
*Cenarion Helm was upgraded with level 1 stamina and level 1 agility. I magnetized a leather helmet (Ghostwalker Crown) on to it. The helmet has 10 stamina and 10 agility, so the Cenarion Helm should have 11 stamina and 11 agility after the magnetization but instead it has only 1 agility.
